### ISSUE TITLE
Scope Variable to Runbook process

### ISSUE THREAD

**Issue by pemaxim** [OctopusDeployLabs/terraform-provider-octopusdeploy#361]
We need to scope OctopusDeploy variable to Runbook process (screenshot). It can be done only via OctopusDeploy UI. The provider does not support it at the moment.

E.g.:

```
resource "octopusdeploy_variable" "myvar" {
  ...
  scope {
    processes = ["Runbooks-123"]
  }

```

![image](https://user-images.githubusercontent.com/308489/160799758-14f91f75-d320-497e-a62c-9148d5f6569d.png)

**Comment by Justin-Walsh** [comment]
Additional request: https://octopus.zendesk.com/agent/tickets/105930 [internal link]

**Comment by mohamed-haidara-cko** [comment]
We are also looking for this feature.

**Comment by haskins60** [comment]
Same here, would love to see this issue resolved....

**Comment by Taha-cmd** [comment]
Same here. This is really important for us.

**Comment by Taha-cmd** [comment]
Hi @zentron 
Sorry for mentioning you directly, but this issue is open for more than a year now and there is an open pull request for it. By inspecting the commit history, I assume that you are a maintainer who can move things forward. Can you give us an ETA on this missing feature and when/if it is going be fixed?

**Comment by zentron** [comment]
Thanks @Taha-cmd , ill aim to take a look at this early next week if possible.

**Comment by andrew22224** [comment]
Waiting on this one as well 🤞

**Comment by zentron** [comment]
Apologies for the delay on this one all. Just getting through the holiday slowdown...

**Comment by tleed5** [comment]
👋 Sorry for the delays on this one, the required changes have been merged in and is available in version `v0.14.4`
